dojo/api_v2/permissions.py

```diff
@@ -1077,3 +1079,9 @@ def has_permission(self, request, view):
         return user_has_global_permission(
             request.user, Permissions.Api_v2_Key
         )
+
+
+class IsAPIImporter(permissions.BasePermission):
+    def has_permission(self, request, view):
```

Comment:
validate it with global role of user. If ti has the global role API_Importer or Maintainer